7.3-cli/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM php:7.3-cli
+
+MAINTAINER Andries Seutens <andries@egeniq.com>
+
+RUN apt-get update && apt-get install -y curl wget git zlib1g-dev libzip-dev locales unzip && rm -rf /var/lib/apt/lists/*
+
+# Locales
+COPY assets/locale.gen /etc/locale.gen
+RUN locale-gen
+
+# PHP
+RUN docker-php-ext-configure mysqli --with-mysqli=mysqlnd && \
+    docker-php-ext-install mysqli && \
+    docker-php-ext-configure pdo_mysql --with-pdo-mysql=mysqlnd && \
+    docker-php-ext-install pdo_mysql && \
+    docker-php-ext-install opcache && \
+    docker-php-ext-install zip
+
+COPY ./assets/php-custom.ini /usr/local/etc/php/conf.d/zzz-custom.ini
+
+ONBUILD COPY src /src
+
+WORKDIR /src
+
+ENTRYPOINT ["docker-php-entrypoint"]
```

Comment:
I suggest using the `/entrypoint.sh` as entrypoint as this will install composer dependencies at launch.
That would be consistent with the rest of the images in this family (predictability)
All the other images also have composer+prestissimo installed, why was it left out for this image?

Reply:
Stripped everything i didnt need. Re-added